packages/components/src/components/post-accordion-item/post-accordion-item.tsx

```diff
@@ -55,15 +56,17 @@ export class PostAccordionItem {
     this.validateHeadingLevel();
   }
 
-  // capture to make sure the "collapsed" property is updated before the event is consumed
+  // Capture to make sure the "collapsed" property is updated before the event is consumed
   @Listen('postToggle', { capture: true })
   onCollapseToggle(event: CustomEvent<boolean>): void {
-    if (
-      event.target === this.host &&
-      (event.target as HTMLElement).localName === 'post-accordion-item'
-    ) {
-      this.collapsed = !event.detail;
-    }
+    eventGuard(
+      this.host,
+      event,
+      { targetLocalName: 'post-accordion-item', delegatorSelector: 'post-collapsible' },
```

Comment:
This still seems to be wrong imo.
Is the `post-accordion-item` component really listening to a `postToggle` event, emitted by a `post-accordion-item` element, where its host element (`this.host`) equals an element which is selectable with `post-collapsible`?

In my opinion, we're listening to a `postToggle` event, emitted by a `post-collapsible` element, while its closest parent should be a `post-accordion-item` element that matches `this.host`.